source/Ship.cpp

```diff
@@ -2660,6 +2670,9 @@ double Ship::JumpDriveFuel() const
 	if(!attributes.Get("jump drive"))
 		return 0.;
 	
+	if(HasOutfit("jump drive", "scram drive"))
+		return BestFuel("jump drive", "scram drive", 175.);
```

Comment:
It seems to me like a scram jump drive would be better than a normal jump drive, so I'd expect the default cost to be more than a normal JD, not less. Perhaps 250 instead of 175.

Reply:
I would have thought that too. But in reality it is somewhat better in specific circumstances (jumping into combat you intend to fight) and somewhat worse otherwise: having to align to the destination to jump means that multiple jumps actually takes longer, sometimes a lot longer for slow turning ships, and if you jump into combat you want to escape from, you are more vulnerable because you spend longer in the system aligning to the new destination and take more fire before you jump out. Thus, I think it has some advantages, and some disadvantages. Since the normal case is that you're just jumping a bunch of hops to get somewhere and usually avoiding combat, the scram-enabled jump drive is actually worse than a regular jump drive, this reducing the fuel costs seemed to be a way to balance out the negatives. You take a little longer each jump prep, but use a little less fuel per jump, so people will have to make a choice on which drive they actually prefer. Thoughts?